index.js

```diff
@@ -205,10 +189,6 @@ class IgnitorPlugin {
       setTimeout(invokeRemote(func, this.stage), 1500);
     }
   }
-
-  clean() {
-    utils.rm(this.packageDir);
```

Comment:
Remove this, we can just add the `build.clean` as the function that is triggered during `ignitor:clean:clean`